### PR TITLE
don't load Filesystem when using CalDAV or CardDAV

### PR DESCRIPTION
**Pull request by georgehrke**
Signed-off-by: Georg Ehrke <developer@georgehrke.com>

### CODE REVIEW

File: apps/dav/lib/Server.php [hunk 1]
```diff
@@ -325,4 +290,114 @@ private function requestIsForSubtree(array $subTrees): bool {
...
+			));
+		});
+	}
+
+	/**
+	 * @param \Symfony\Component\EventDispatcher\EventDispatcherInterface $dispatcher
+	 * @param SabrePluginEvent $event
+	 * @param RootCollection $root
+	 */
+	private function loadAppsPlugins(\Symfony\Component\EventDispatcher\EventDispatcherInterface $dispatcher, SabrePluginEvent $event, RootCollection $root) {
```
Comment (ChristophWurst) [inline apps/dav/lib/Server.php]: Use `\OCP\EventDispatcher\IEventDispatcher` ;)

**Comment by tcitworld** [comment]
Related to https://github.com/nextcloud/server/issues/6447 ?

**Comment by georgehrke** [comment]
I think so, yes. Although that was not my intention, when fixing this.

**Comment by ChristophWurst** [comment]
@georgehrke :ping_pong: :)

**Comment by rullzer** [comment]
I'm going to close this due to lack of activity.
Feel free to reopen if anybody wants to continue.

**Pull request closed without merging**